compiler_gym/random_replay.py

```diff
@@ -8,7 +8,7 @@
 
 from deprecated import deprecated
 
-from compiler_gym.envs.compiler_env import CompilerEnv
```

Comment:
IRRC, this is actually useful. It means that `//compiler_gym:random_replay` need only depend on `//compiler_gym/envs:compiler_env` rather than `//compiler_gym`

Reply:
Done